### PR TITLE
BZ:2020128 - Showing that OpenShift 3.11 is on vSphere 7

### PR DESCRIPTION
**Pull request by kelbrown20**
For Version 3.11
Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=2020128

Preview: https://deploy-preview-38561--osdocs.netlify.app/openshift-enterprise/latest/install_config/configuring_vsphere#vsphere-prereqs

Ready for QA: @gpei 

For peer reviewers: Can you please add the 'enterprise-3.11' label, thank you!

### CODE REVIEW

File: install_config/topics/vsphere_prerequisites.adoc [hunk 1]
```diff
@@ -16,6 +16,11 @@ Standalone ESXi is not supported.
 * vSAN, VMFS and NFS supported.
 ** vSAN support is limited to one cluster in one vCenter.
 
+[NOTE]
+====
+OpenShift 3.11 is supported and deploys on vSphere 7 clusters. Storage options vSAN, VMFS and NFS are also supported with the vSphere in-tree storage driver.
```
Comment (kalexand-rh) [inline install_config/topics/vsphere_prerequisites.adoc:21]: Unless you're talking about a specific subcomponent that uses "OpenShift" in its name, "{product-title}" is basically always more correct than "OpenShift." I also suggest flipping the second sentence around. 

```suggestion
{product-title} 3.11 is supported and deploys on vSphere 7 clusters. If you use the with the vSphere in-tree storage driver, vSAN, VMFS, and NFS storage options are also supported. 
```

**Comment by gpei** [comment]
Hi , I can ack this `3.11 is supported and deploys on vSphere 7 clusters`, but for the following words:
`If you use the vSphere in-tree storage driver, vSAN, VMFS and NFS storage options are also supported`, maybe we need @jhou1 or @duanwei33 's review.

**Comment by kelbrown20** [comment]
Thank you! @jhou1 or @duanwei3, can you take a look at this and conform the info?

**Comment by duanwei33** [comment]
Looks good to me (According to https://access.redhat.com/solutions/6075031)
@gnufied Could you also take a look here?

**Pull request merged**